### ISSUE TITLE
License incorrectly set

### ISSUE THREAD

**Issue by mcaccin** [rspeer/python-ftfy#218]
According to the [pyproject.toml specifications](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license), this line 
https://github.com/rspeer/python-ftfy/blob/38b966756ffd348954621be1817faa152a8b7851/pyproject.toml#L9
should be either 
```toml
license = {text = "Apache-2.0"}
```
or 
```toml
license = {file = "LICENSE.txt"}
```
With the newest release 6.3.0, the license isn't shown on PyPi and a tool like [pip-licenses](https://github.com/raimon49/pip-licenses) doesn't pick it up either.

**Comment by rspeer** [comment]
I'll resolve that in the next release, particularly because the license terms are important to me

but maybe they should fix that? I can't be the only person porting from Poetry to the new standard, and there was no warning of any kind about the existing format being wrong

**Comment by mcaccin** [comment]
Thanks for the quick reply, and yes I understand the importance of a license, that's why I happen to notice the problem in the first place :) 
Regarding the second paragraph, I don't have experience with poetry but it looks like it's using its own syntax which differs in random places with the vanilla pyproject.toml specification (they get away with it with the [tool.poetry] section header), whereas uv stays true to the original documentation. So the "they" is a bit undefined... For completeness, [RUF200](https://docs.astral.sh/ruff/rules/invalid-pyproject-toml/) doesn't seem to catch this either, so yeah looks like it's a loaded footgun for many

**Comment by rspeer** [comment]
Thanks for raising that!

I fixed it in 6.3.1 and checked that it's showing correctly on PyPI.
